ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -430,4 +430,27 @@ def get_deviceset_count():
     sc = get_storage_cluster()
     return int(sc.get().get('items')[0].get('spec').get(
         'storageDeviceSets')[0].get('count')
+               )
```

Comment:
plz fix indentation

Reply:
not a part of the PR

Reply:
1. it keep failing flaks8 ci, so you have to fix it
2. it is different than how it is in master, so i might changed without intention (automatic formatting etc.)